WolvenKit.CLI/Commands/SettingsCommand.cs

```diff
@@ -19,12 +20,31 @@ private void Action(IHost host)
         var basedir = AppDomain.CurrentDomain.BaseDirectory;
         var settingsPath = Path.Combine(basedir, "appsettings.json");
         Console.WriteLine($"Opening {settingsPath}");
-        new Process
-        {
-            StartInfo = new ProcessStartInfo(settingsPath)
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
+            new Process
             {
-                UseShellExecute = true
+                StartInfo = new ProcessStartInfo(settingsPath)
+                {
+                    UseShellExecute = true
+                }
+            }.Start();
+        } else {
+            var editorCommand = System.Environment.GetEnvironmentVariable("EDITOR");
```

Comment:
I don't have the EDITOR command setup by default on my distro, but as an alternative on linux we could use `xdg-open` which is the closest to a standard.

Reply:
`xdg-open` is what I think is used by default for when you use `Process.Start(filename)` so I think instead of throwing an exception I can just try that. I was trying to follow what GitHub CLI does which is that it prompts you to re-run the command prefixed with `EDITOR=vim` or something. But I think it would be best to just do the same as what it does for Windows if no EDITOR env var is found.

Reply:
Actually yeah I think it might be better to use `xdg-open` than starting the process with the file directly.

Reply:
Wait, xdg-open is Xorg specific and won't work headlessly. Quite the dilemma.

Reply:
It's not Xorg specific (also works with Wayland, and I don't know of any desktop environment which doesn't support it), but you're right about headless.
Maybe it's overcomplicating it, and EDITOR is simple enough

Reply:
My mistake, I think EDITOR may be best with simply opening the file itself as a process as a backup.